internal/tests/testrepository/header_testrepository.go

```diff
@@ -190,23 +190,43 @@ func (r *HeaderTestRepository) GetChainBetweenTwoHashes(low string, high string)
 // GetMerkleRootsConfirmations returns a confirmation of merkle roots inclusion
 // in the longest chain with hash of the block in which the merkle root is included.
 func (r *HeaderTestRepository) GetMerkleRootsConfirmations(
-	merkleroots []string,
+	request []domains.MerkleRootConfirmationRequestItem,
+	maxBlockHeightExcess int,
 ) ([]*domains.MerkleRootConfirmation, error) {
 	mrcfs := make([]*domains.MerkleRootConfirmation, 0)
 
-	for _, mr := range merkleroots {
-		hash := ""
+	var topHeight int32 = 0
```

Comment:
Shouldn't we use init style like `topHeight:= int32(0)` ?

Reply:
True, done